README.md

```diff
@@ -10,7 +10,17 @@ Quick two truths and a lie:
 
 3. I ran a personal best in the 5k this weekend
 
+Here is a [**link**](https://www.linkedin.com/in/jarretangbazo) to my LinkedIn page. There you will be able to find my employment and education histories.
+
+Thanks for stopping by.
+
+### Process
+To get the changes onto Readme.md, I essentially followed the instructions from the course webpage [**Using Git within RStudio**](https://cfss.uchicago.edu/setup/git-with-rstudio/). Initially, everything did not install correctly so I had to manually change the Git executable path from **/usr/bin/git** to **/usr/local/bin/git**. I then followed the steps for forking and cloning a repository, making edits to the readme file, saving and pushing back to GitHub.
+
+### Reflection on assignment
+To complete this assignment, I used the following [**link**](https://www.markdownguide.org/cheat-sheet/) as a resource for the markdown syntax.
+
+I still feel fairly uncomfortable with Github and perhaps this is the main reason why I am taking this class. The process of forking, cloning, pulling, and pushing seems very complicated. While I am aware that this is mostly due to the terminology associated with GitHub and I'm certain in time this should get easier, it all seems very opaque. Even though I've completed this assignment, I'm not completely sure that I've understood what I've done completely.
```

Comment:
I also feel uncomfortable Github. It was helpful to work with other students in the lab session and ask questions in a smaller setting. Although it was hard to get the attention of the TA and professor, the lab session is a great space for collaboration.